### DISCUSSION TITLE
Make dropdown and radio field defaults overridable

### DISCUSSION THREAD

**Discussion by gateszies** [craftcms/cms#15661] [Ideas] [answered]
Bouncing off [this idea](https://github.com/craftcms/cms/discussions/15660), it'd be great if you could override the default values for dropdown and radio fields on an as-needed basis. We often re-use the same dropdown or radio values but with different defaults defined, ie. needing the default `Padding Top` to be `none` on one block and `large` on another. This currently requires creating duplicate fields just to re-define the default.
<img width="500" alt="Screenshot 2024-09-04 at 9 33 23 AM" src="https://github.com/user-attachments/assets/f4682744-e9a4-45a7-ac1c-28c4543f5f1c">

Or needing `left` as the default on one block and `right` as the default on another. Currently, we have to create two separate fields to achieve this.
<img width="500" alt="Screenshot 2024-09-04 at 9 33 11 AM" src="https://github.com/user-attachments/assets/7d46221b-28c0-43f2-a9b3-4725681c95b1">

**Comment by gateszies** [comment, marked as answer]
Looks like some similar discussions were happening over [here](https://github.com/craftcms/cms/discussions/8688)

**Reply by brandonkelly** [reply]
Yeah, just pointed #15660 to that discussion as well. I’m not sure how we’d go about partially overriding the options / hiding certain options for Dropdown fields on a per-instance basis though.

**Reply by ryanleichty** [reply]
This is also very similar too https://github.com/craftcms/cms/discussions/15312
